research/cifar10/ditto/run_fold_experiment.slrm

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
+
+#SBATCH --nodes=1
+#SBATCH --ntasks=1
+#SBATCH --ntasks-per-node=1
+#SBATCH --cpus-per-task=1
+#SBATCH --gres=gpu:1
+#SBATCH --mem=32G
+#SBATCH --partition=a40
```

Comment:
True I think it would fit but on the other hand I have no issues getting a40s allocated 🤣 But you are right we should be mindful of our resource usage.

Reply:
Fair enough. Just a suggestion. If you've had no trouble then I don't mind either way